### PR TITLE
[Nataliia Falinska] MoGo-JS-Task3

### PR DESCRIPTION
**Pull request by NFalinska**

### CODE REVIEW

**Comment by NFalinska** [comment]
https://nfalinska.github.io/ui-internship-lwo2/css-practice/task_3/index.html

**Comment by dverbovyi** [comment]
I've found several issues, 

1 - image opened not in true popup, because header is overlapped and I have ability to scroll the page content
screen shot 2018-12-09 at 23 07 16

2. - Accordion tab opens with animation but closing without it. It's a rough inconsistency which should be fixed

3 - Link blog scrolls to incorrect section "What people say" instead of "Our stories latest blog"

4 - Jumpiness of content and broken header, please see the video https://drive.google.com/file/d/1aYHwC1aa8WRwfYeiryB8g8agxJq4rd0o/view?usp=sharing

pls, reach me out in DM for further explanation if you need it

File: css-practice/task_3/accordion.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const accHead = document.getElementsByClassName('accordion-item-heading');
+for (let i = 0; i < accHead.length; i++) {
+  accHead[i].addEventListener('click', toggleItem, false);
+}
+function toggleItem() {
+  let itemClass = this.parentNode.className;
+  for (let i = 0; i < accItem.length; i++) {
+    accItem[i].className = 'accordion-item close';
+  }
+  if (itemClass == 'accordion-item close') {
```
Comment (dverbovyi) [inline css-practice/task_3/accordion.js]: use classList.contains('close')

File: css-practice/task_3/accordion.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/* global document */
+const accItem = document.getElementsByClassName('accordion-item');
```
Comment (dverbovyi) [inline css-practice/task_3/accordion.js]: don't write code in global scope

**Comment by dverbovyi** [comment]
@NFalinska 
screen shot 2018-12-10 at 19 47 32

jumpiness isn't fixed 4 - Jumpiness of content and broken header, please see the video https://drive.google.com/file/d/1aYHwC1aa8WRwfYeiryB8g8agxJq4rd0o/view?usp=sharing

File: css-practice/task_3/slider.js [hunk 3]
```diff
@@ -1,26 +1,60 @@
...
-  carouselSlideOne.style.transform = 'translateX(' + (-size * counterOne) + 'px)';
-});
-
-prevBtnOne.addEventListener('click', () => {
-  if (counterOne <= 0) return;
-  carouselSlideOne.classList.add('transition');
-  counterOne--;
-  carouselSlideOne.style.transform = 'translateX(' + (-size * counterOne) + 'px)';
-});
+// /* eslint-disable max-len */
```
Comment (dverbovyi) [inline css-practice/task_3/slider.js]: don't commit commented code!

File: css-practice/task_3/slider.js [hunk 4]
```diff
@@ -1,26 +1,60 @@
...
+  const rightArrow = slideEl.querySelector('.arrow-right');
+  const scroller = slideEl.querySelector('.scroller');
+
+
+  let counterOne = 0;
+
+  scroller.style.transform = 'translateX(' + (-98 * counterOne) + '%)';
+
+  leftArrow.addEventListener('click', () => {
+    //  if (counterOne >= carouselImagesOne.length - 1) return;
```
Comment (dverbovyi) [inline css-practice/task_3/slider.js]: don't commit commented code

File: css-practice/task_3/header.js [hunk 5]
```diff
@@ -1,7 +1,6 @@
 /* global document, window, pageYOffset */
-
+const header = document.getElementsByTagName('nav')[0];
```
Comment (dverbovyi) [inline css-practice/task_3/header.js]: global scope variable detected!

File: css-practice/task_3/header.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+/* global document, window, pageYOffset */
+
+window.addEventListener('scroll', function() {
+  const header = document.getElementsByTagName('nav')[0];
```
[the remark this answers is not in the document]
Comment (dverbovyi) [reply css-practice/task_3/header.js]: @NFalinska this comment hasn't been solved

File: css-practice/task_3/accordion.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+function changeTabs(event) {
+  if (event.target.className == 'accordion-item-heading') {
+    const dataTab = event.target.getAttribute('data-tab');
+    const tabBody = document.getElementsByClassName('tab-b');
+    for (let i = 0; i < tabBody.length; i++) {
+      if (dataTab == i) {
+        tabBody[i].style.display = 'block';
+      } else {
+        tabBody[i].style.display = 'none';
```
Comment (dverbovyi) [inline css-practice/task_3/accordion.js]: duplication

try to rewrite in such way:
```
tabBody[i].style.display = dataTab === i ? 'block' : 'none'
```

File: css-practice/task_3/popup.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+/* global document */
+
+const modal = document.getElementById('pic-modal');
```
Comment (dverbovyi) [inline css-practice/task_3/popup.js]: global scope variables

**Pull request merged**